libs/webb-ui-components/src/index.ts

```diff
@@ -1,3 +1,5 @@
+'use client';
+
```

Comment:
This is okay, but I propose we create a task to review our components in the `webb-ui-kit`. The aim would be to determine which components are `client`-related, and which are associated with the `server`. Could you kindly create an issue to address this?